functions/identifier.js

```diff
@@ -0,0 +1,10 @@
+const { t, src } = require('../internal/re')
+const identifier = (version) => {
+  const split = version.split('.')
+  const valid = split.every(part => new RegExp(`^${src[t.PRERELEASEIDENTIFIER]}$`).test(part))
```

Comment:
For performance reasons, is better to cache this RegExp instead of creating an instance every time.

Reply:
I agree. This should exist in `internal/re.js`.